submissions/Nemooochka/tiny-js-upd/index.js

```diff
@@ -0,0 +1,126 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(species, name, gender, legs, saying, friends = []) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.saying = saying;
+        this.friends = friends;
+        this.addFriends(friends);
```

Comment:
Here we got direct assignment of `.friends` with the value from constructor params, then again assignment to `.friends` by calling `.addFriends`.
Looks a bit redundant. A vote for assignment via the method as it is created for purpose and encapsulates all required logic to properly handle the received argument.

But if we just remove direct `.friends` assignment then the logic inside `.addFriends` may break.

Two options with this problem:
(1) either assign `.friends` with empty array in constructor
 or
(2) handle creation of `.friends` in `.addFriends` if the property doesn't exist yet

It is up to you to decide what approach is cleaner, SOLID, DRY etc

Yet your